templates/institutional-landing/institutional-landing-fr.html

```diff
@@ -0,0 +1,120 @@
+			</div>
+			<div class="col-lg-4 col-md-6">
+				<h3><a href="#">[Lien hypertexte]</a></h3>
+				<p>Utilisez des verbes d'action ou répertoriez simplement des mots-clés pour résumer les informations ou les tâches qui peuvent être accomplies sur la page à laquelle elles renvoient.</p>
+			</div>
+			<div class="col-lg-4 col-md-6">
+				<h3><a href="#">[Lien hypertexte]</a></h3>
+				<p>Utilisez des verbes d'action ou répertoriez simplement des mots-clés pour résumer les informations ou les tâches qui peuvent être accomplies sur la page à laquelle elles renvoient.</p>
+			</div>
+			<div class="col-lg-4 col-md-6">
+				<h3><a href="#">[Lien hypertexte]</a></h3>
+				<p>Utilisez des verbes d'action ou répertoriez simplement des mots-clés pour résumer les informations ou les tâches qui peuvent être accomplies sur la page à laquelle elles renvoient.</p>
+			</div>
+		</div>
+	</section>
+</div>
+
+<div class="well well-sm brdr-0 mrgn-bttm-md">
+	<div class="container">
+		<section>
+			<h2>Contactez-nous</h2>
+			<ul class="wb-eqht list-unstyled mrgn-tp-lg lst-spcd-2 colcount-md-3 bold-content">
+				<li><a href="#">Contactez [Institution]</a></li>
+				<li><a href="#">[Tâche de contact principale 2]</a></li>
+				<li><a href="#">[Tâche de contact principale 3]</a></li>
+			</ul>
+		</section>
+	</div>
+</div>
+
+<div class="container">
+	<div class="row">
+		<div class="col-md-8">{%- include /design-patterns/gc-institution/gc-institution.html -%}</div>
+		<div class="col-md-4">{%- include gc-follow-us/gc-follow-us.html -%}</div>
+	</div>
+
+	{%- include /components/gc-minister/gc-minister-two-ministers.html -%}
+
+	<div class="row">
+		<div class="col-md-4">
+			<section class="wb-feeds limit-3">
+				<h2>Nouvelles</h2>
+				<ul class="feeds-cont list-unstyled lst-spcd">
+					<li><a data-ajax="https://wet-boew.github.io/GCWeb/components/feeds/demo/example-en.atom.json" href="#">Nouvelles de [Institution]</a></li>
+				</ul>
+				<p>Plus: <a href="#">nouvelles de [Institution]</a></p>
+			</section>
+		</div>
+
+		<div class="col-md-8">
+			<section class="gc-features">
+				<h2>En vedette</h2>
+				<div class="row wb-eqht-grd">
+					<div class="col-sm-6">
+						<div class="well well-sm brdr-rds-0 eqht-trgt">
+							<img class="img-responsive full-width" src="https://via.placeholder.com/360x203/000000/FFFFFF.png" alt="" />
+							<h3 class="h5"><a class="stretched-link" href="#">[Lien à l'élément en vedette]</a></h3>
+							<p>Brève description de l'élément en vedette.</p>
+						</div>
+					</div>
+					<div class="col-sm-6">
+						<div class="well well-sm brdr-rds-0 eqht-trgt">
+							<img class="img-responsive full-width" src="https://via.placeholder.com/360x203/000000/FFFFFF.png" alt="" />
+							<h3 class="h5"><a class="stretched-link" href="#">[Lien à l'élément en vedette]</a></h3>
+							<p>Brève description de l'élément en vedette.</p>
+						</div>
+					</div>
+				</div>
+			</section>
```

Comment:
Use the code sample to be included instead

```suggestion
		{%- include components/gc-features/features.html -%}
```